google/src/com/news/searcher/service/Executor.java

```diff
@@ -0,0 +1,66 @@
+package com.news.searcher.service;
+
+import com.news.searcher.model.LFUCache;
+import com.news.searcher.repository.Repository;
+
+import java.util.*;
+
+public class Executor {
+    private Scanner scanner;
+    private LFUCache lfuCacheList;
+    private List<String> allEvents;
+    private List<String> checkList;
+
+    public Executor() {
+        Repository repository = new Repository();
+        this.lfuCacheList = repository.getLfuList();
+        this.allEvents = repository.getAllEvents();
+        this.checkList = repository.getCheckList();
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void startApp() {
+        printAllEventsList();
+        startSelectingEventsFromConsole();
+        printFLUCacheListIntoConsole();
+    }
+
+    private void printAllEventsList() {
+        for (int i = 0; i < allEvents.size(); i++) {
```

Comment:
maybe foreach?

Reply:
probably yes, but for this case I need counter to mark the events

Reply:
> maybe foreach?

probably yes, but for this case I need counter to mark the events